File: Spring Week 2/templates/B/B_jollyjumpers.py
import sys

# Input: N and A, where A is a list of N integers
# Output: True if the list is Jolly, False otherwise
def jollyjumpers(N,A):
    #Your code here
    return True

for line in sys.stdin:
    if(line == "#\n") : break # You can manually end input with a '#'
    line_ints = list(map(int, line.split()))
    N = line_ints[0]
    A = line_ints[1:]
    print("Jolly" if jollyjumpers(N,A) else "Not jolly")

File: Spring Week 2/templates/B/B_jollyjumpers.cpp
#include <bits/stdc++.h>
using namespace std;

// Input: N and A, where A is a vector of N integers
// Output: True if the vector is Jolly, False otherwise
bool jollyjumpers(int N, vector<int> A){
    //Your code here
    return true;
}

int main(){
    int N;
    while(cin >> N){
        vector<int> A(N);
        for(int i = 0; i < N; i++){
            cin >> A[i];
        }
        cout << (jollyjumpers(N,A) ? "Jolly" : "Not jolly") << endl;
    }
}